index.js

```diff
@@ -1 +1 @@
w=await a.getDiff(b,g);_.startGroup("Size Differences:");const v=await a.printSizes(w);console.log(v),_.endGroup();const E=function(e,{showTotal:t,collapseUnchanged:r,omitUnchanged:o,minimumChangeThreshold:n}){let s=[],i=[],a=0,c=0;for(const t of e){const{filename:e,size:u,delta:l}=t;a+=u,c+=l;const p=u-l,f=Math.abs(l)<n;if(f&&o)continue;const d=[`\`${e}\``,Ls(u),hu(l,p),mu(l,p)];f&&r?i.push(d):s.push(d)}let u=gu(s);if(0!==i.length&&(u+=`\n\n<details><summary>ℹ️ <strong>View Unchanged</strong></summary>\n\n${gu(i)}\n\n</details>\n\n`),t){const e=a-c;let t=hu(c,e),r=mu(c,e);u=`**Total Size:** ${Ls(a)}\n\n${u}`,u=`**Size Change:** ${t} ${r}\n\n${u}`}return u}(w,{collapseUnchanged:yu(_.getInput("collapse-unchanged")),omitUnchanged:yu(_.getInput("omit-unchanged")),showTotal:yu(_.getInput("show-total")),minimumChangeThreshold:parseInt(_.getInput("minimum-change-threshold"),10)});let T=!1;const S={...r.repo,issue_number:n},O=_.getInput("comment-key"),k={...S,body:E+`\n\n<a href="https://github.com/preactjs/compressed-size-action"><sub>compressed-size-action${O?"::"+O:""}</sub></a>`};if("pull_request"!==r.eventName&&"pull_request_target"!==r.eventName)console.log("No PR associated with this action run. Not posting a check or comment."),T=!1;else if(yu(_.getInput("use-check")))if(o){const t=await async function(e,t){const r=await e.checks.create({...t.repo,name:"Compressed Size",head_sha:t.payload.pull_request.head.sha,status:"in_progress"});return async o=>{await e.checks.update({...t.repo,check_run_id:r.data.id,completed_at:(new Date).toISOString(),status:"completed",...o})}}(e,r);await t({conclusion:"success",output:{title:"Compressed Size Action",summary:E}})}else T=!0;else{let t;_.startGroup("Updating stats PR comment");try{const r=(await e.issues.listComments(S)).data,o=new RegExp(`<sub>[s\n]*(compressed|gzip)-size-action${O?"::"+O:""}</sub>`);for(let e=r.length;e--;){const n=r[e];if(o.test(n.body)){t=n.id;break}}}catch(e){console.log("Error checking for previous comments: "+e.message)}if(t){console.log("Updating previous comment #"+t);try{await e.issues.updateComment({...r.repo,comment_id:t,body:k.body})}catch(e){console.log("Error editing previous comment: "+e.message),t=null}}if(!t){console.log("Creating new comment");try{await e.issues.createComment(k)}catch(t){console.log("Error creating comment: "+t.message),console.log("Submitting a PR review comment instead...");try{const t=r.issue;await e.pulls.createReview({owner:t.owner,repo:t.repo,pull_number:t.number,event:"COMMENT",body:k.body})}catch(e){console.log("Error creating PR review."),T=!0}}}_.endGroup()}T&&console.log(`\n\t\t\tError: compressed-size-action was unable to comment on your PR.\n\t\t\tThis can happen for PR's originating from a fork without write permissions.\n\t\t\tYou can copy the size table directly into a comment using the markdown below:\n\t\t\t\n\n${k.body}\n\n\n\t\t`.replace(/^(\t|  )+/gm,"")),console.log("All done!")}(r,_t.context,e)}catch(e){_.setFailed(e.message)}})();
```

Comment:
Please revert, we'll change this when cutting a release

Reply:
Done